src/main/java/dev/tourmi/svmm/server/Tunneler.java

```diff
@@ -83,7 +84,16 @@ public static ClientStatus doTunnel(Player player, Level level, ItemStack heldIt
 
         status.lastBlocksMined = blocks.size();
         status.lastPosition = blockPos;
-        player.sendSystemMessage(Component.literal(String.format("Mined %d blocks", status.lastBlocksMined)));
+        for (Method m : Player.class.getMethods()) {
+            if (m.getName().contains("send") || m.getName().contains("message")) {
+                System.out.println(m.toString());
+            }
+        }
+
+		if (!level.isClientSide() && player instanceof ServerPlayer serverPlayer) {
+			serverPlayer.sendSystemMessage(Component.literal("Message"));
+		}
+
```

Comment:
You can revert all of this, updating the function signature should be enough to fix the compilation errors